docs/tutorials/articles/dataiku/index.md

```diff
@@ -187,36 +106,96 @@ Create your own Taipy application to visualize all this information. It can
+    md = """
+    # Dataiku **Overview**{: .color-primary}
 
-**Projects**
+    **Projects**
 
-<|{selected_projects}|selector|lov={list(metrics_df['Project'].unique())}|dropdown|multiple|class_name=fullwidth|label=Projects|>
+    <|{selected_projects}|selector|lov={list(metrics_df['Project'].unique())}|dropdown|multiple|class_name=fullwidth|label=Projects|>
 
-**Metrics**
+    **Metrics**
 
-<|{selected_metrics}|selector|lov={list(metrics_df['MetricType'].unique())}|dropdown|multiple|class_name=fullwidth|label=Metrics|>
+    <|{selected_metrics}|selector|lov={list(metrics_df['MetricType'].unique())}|dropdown|multiple|class_name=fullwidth|label=Metrics|>
 
-<|{show_projects(metrics_df, selected_projects, selected_metrics)}|table|filter|>
+    <|{show_projects(metrics_df, selected_projects, selected_metrics)}|table|filter|>
 
-<|{show_projects(metrics_df, selected_projects, selected_metrics)}|chart|x=Name|y=Value|type=bar|>
-"""
+    <|{show_projects(metrics_df, selected_projects, selected_metrics)}|chart|x=Name|y=Value|type=bar|>
+    """
 
-Gui(md).run()
-```
+    Gui(md).run()
+    ```
+
+    [Download the code](./src/metrics_visualization.py){: .tp-btn target='blank' }
+
+=== "Python"
+    ```python
+    from taipy.gui import Gui
+    import taipy.gui.builder as tgb
+
+    selected_projects = list(metrics_df['Project'].unique())
+    selected_metrics = ["AUC"]  # Initially focusing on AUC metrics.
+
+    def show_projects(metrics_df, selected_projects, selected_metrics):
+        return metrics_df[(metrics_df['Project'].isin(selected_projects)) &
+                        (metrics_df['MetricType'].isin(selected_metrics))]
+
+    with tgb.Page() as page:
+        tgb.text("# Dataiku **Overview**", mode="md")
+
+        tgb.text("**Projects**", mode="md")
+
+        tgb.selector(
+            "{selected_projects}",
+            lov=list(metrics_table["Project"].unique()),
+            multiple=True,
+            label="Projects",
+            dropdown=True,
+            class_name="fullwidth",
+        )
+
+        tgb.text("**Metrics**", mode="md")
+
+        tgb.selector(
+            "{selected_metrics}",
+            lov=list(metrics_table["MetricType"].unique()),
+            multiple=True,
+            label="Metrics",
+            dropdown=True,
+            class_name="fullwidth",
+        )
+
+        tgb.table(
+            lambda metrics_table, selected_projects, selected_metrics: show_projects(
+                metrics_table, selected_projects, selected_metrics
+            ),
+            filter=True,
+        )
+
+        tgb.chart(
+            lambda metrics_table, selected_projects, selected_metrics: show_projects(
+                metrics_table, selected_projects, selected_metrics
+            ),
+            x="Name",
+            y="Value",
+            type="bar",
+        )
+
+    Gui(md).run()
+    ```
+
+    [Download the code](./src/metrics_visualization.py){: .tp-btn target='blank' }
```

Comment:
One last thing, this link is wrong, it should link to the tgb version

Reply:
Thanks